packages/adblocker/src/resources.ts

```diff
@@ -418,9 +424,7 @@ export default class Resources {
     for (const { name, aliases, body: content, contentType } of this.resources) {
       buffer.pushASCII(name);
       buffer.pushUint16(aliases.length);
-      for (const alias of aliases) {
-        buffer.pushASCII(alias);
-      }
+      aliases.forEach((alias) => buffer.pushASCII(alias));
```

Comment:
Nit. I don't see this necessarily as an improvement given we use `for .. of` in other places.